### DISCUSSION TITLE
Tracking the picks in excel

### DISCUSSION THREAD

**Discussion by Fastlane86** [kyleskom/NBA-Machine-Learning-Sports-Betting#318] [General]
Has anyone started a project yet with tracking the picks in excel or google docs? Since its the beginning of the season. Was just curious. If no one has I'm planning on starting one over the weekend. I have the picks saved from the past couple days. So I'll back log it.

**Comment by Skysafe64** [comment]
I'm already on it.

**Reply by jnish23** [reply]
Awesome, I was also looking for this. Would either of you mind sharing a google sheet? I would be happy to help inputting results. It would also be nice to eventually automate this process to track results as the season progresses. 
Thanks!

**Reply by Baaios** [reply]
Do you track using Colab results or locally on Python? I run from Colab and my results since now on ML are 60%.

**Comment by Skysafe64** [comment]
> Do you track using Colab results or locally on Python? I run from Colab and my results since now on ML are 60%.

colab. 60% success means absolutely nothing in reality. you can even have 80% and be in deficit.

since the update, some results are difficult to understand

**Reply by handleygf** [reply]
This. Oddly from what I'm tracking, if you were to bet only on games where the predicted winner has a negative EV according to the model, it would be up 8% ROI. (as of 10/31)

Betting only positive EV predicted winners is getting hammered, down -14% ROI

**Comment by Skysafe64** [comment]
> Awesome, I was also looking for this. Would either of you mind sharing a google sheet? I would be happy to help inputting results. It would also be nice to eventually automate this process to track results as the season progresses. Thanks!

working on it. I'll let you know

**Reply by BitcoinGPT** [reply]
Any updates? I wouldn’t mind adding some insights as well, glad to actually see there are some warriors attacking this

**Comment by Skysafe64** [comment]
> This. Oddly from what I'm tracking, if you were to bet only on games where the predicted winner has a negative EV according to the model, it would be up 8% ROI. (as of 10/31)
> 
> Betting only positive EV predicted winners is getting hammered, down -14% ROI

That's fun, idk what to say. It was perfect last season for values but now ...
